lib/index.js

```diff
@@ -63,19 +65,31 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   app.get(
     basePath,
     optionallyAuthenticated,
-    //validate({query: 'services.account.getAccountsQuery'}),
-    asyncHandler(async (req, res, next) => {
-      if(req.query.exists === 'true' && req.query.email) {
+    // FIXME add an implementation in bedrock-express
+    boolParser(),
+    validate({query: 'account.getAccountsQuery'}),
+    asyncHandler(async (req, res) => {
+      if(req.query.exists === true && req.query.email) {
         // allow anyone to check for account existence
         const {email} = req.query;
         const exists = await brAccount.exists({actor: null, email});
         if(exists) {
           return res.status(200).end();
         }
-        return res.status(404).end();
+        // TODO: improve error details
+        throw new BedrockError(
+          'Account does not exist.', 'NotFoundError', {
+            httpStatusCode: 404,
+            public: true,
+          });
       }
       // TODO: call `brAccount.getAll`
-      next();
+      // TODO: improve error details
+      throw new BedrockError(
```

Comment:
This seems odd to change this to an account not found error ... but since we're about to implement adding `getAll` here I suppose it doesn't matter.

Reply:
incorrect actually or maybe I'm wrong, but I actually implemented the getAll on an admin route I did not re-use this route, but it's less secure.

Reply:
anyways this is more of an issue for the next PR i.e. the get all accounts by email, because we might want to re-use this route with a different query string rather than create a new one.